addons/mail/static/src/core/common/chat_hub.xml

```diff
@@ -8,7 +8,7 @@
         <t t-foreach="visibleChatWindows" t-as="cw" t-key="cw.localId">
             <ChatWindow chatWindow="cw" t-if="cw.canShow" right="chatHub.BUBBLE_START + chatHub.BUBBLE + (chatHub.BUBBLE_OUTER*2) + (visibleChatWindows.length - cw_index - 1) * (chatHub.WINDOW + chatHub.WINDOW_INBETWEEN * 2)"/>
         </t>
-        <div class="o-mail-ChatHub-bubbles position-fixed d-flex flex-column align-content-start justify-content-end align-items-center" t-attf-style="top: {{position.top}}; left: {{position.left}}; right: {{position.right}}; bottom: {{position.bottom}}" t-att-class="{ 'o-liftUp': busMonitoring.hasConnectionIssues, 'o-mobile': isMobileOS }" t-ref="bubbles">
+        <div t-if="!store.discuss?.isActive" class="o-mail-ChatHub-bubbles position-fixed d-flex flex-column align-content-start justify-content-end align-items-center" t-attf-style="top: {{position.top}}; left: {{position.left}}; right: {{position.right}}; bottom: {{position.bottom}}" t-att-class="{ 'o-liftUp': busMonitoring.hasConnectionIssues, 'o-mobile': isMobileOS }" t-ref="bubbles">
```

Comment:
Some AI chat windows and chat bubbles are intended to be shown inside discuss app. That's why the template was changed in 18.4 with `cw.canShow` and `chatHub.showConversations`.

Fix should limit to mobile and should include when messaging menu is open (the discuss app and messaging menu are 90% similar, the 10% difference is poor UI/UX that we plan to solve soon in v19.0). I'd advice to put changes in code of `canShow` and `showConversations` fields, the template should not have any change since it has been designed for visibility based on those fields.